composites/Plugin/Shared/components/SEOScoreAssessment.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const SEOScoreAssessmentItem = styled.li`
+	display:   table-row;
+	font-size: 14px;
+`;
+
+const SEOScoreAssessmentPart = styled.span`
+	display: table-cell;
+	padding: 2px;
+`;
+
+const SEOScoreAssessmentBullet = styled.span`
```

Comment:
This could be extracted to a shared component since it can be reused for the content analysis component.